Python/optimizer_bytecodes.c

```diff
@@ -529,6 +529,7 @@ dummy_func(void) {
 
     op(_LOAD_SMALL_INT, (-- value)) {
         PyObject *val = PyLong_FromLong(this_instr->oparg);
```

Comment:
This can be simplified too, while we're here:

```suggestion
        PyObject *val = PyLong_FromLong(oparg);
```